OpenRA.Mods.Common/Widgets/ObserverSupportPowerIconsWidget.cs

```diff
@@ -193,7 +193,7 @@ public override void Tick()
 				lastIconIdx = i;
 				TooltipIcon = supportPowerIconsIcons[i];
 				currentTooltipToken = tooltipContainer.Value.SetTooltip(TooltipTemplate,
-					new WidgetArgs() { { "world", worldRenderer.World }, { "player", GetPlayer() }, { "getTooltipIcon", GetTooltipIcon } });
+					new WidgetArgs() { { "world", worldRenderer.World }, { "player", GetPlayer() }, { "getTooltipIcon", GetTooltipIcon }, { "playerResources", GetPlayer().PlayerActor.Trait<PlayerResources>() } });
```

Comment:
The trait reference isn't going to change. Querying every tick for every power isn't needed!